torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -331,6 +331,15 @@ CompiledFusionFunction::CompiledFusionFunction(const std::string & name, Annotat
   JIT_CUDA_CHECK(cudaGetDevice(&device));
   JIT_CUDA_CHECK(cudaGetDeviceProperties(&prop, device));
 
+  if ((prop.major >= 6 && CUDA_VERSION < 8000) ||
+      (prop.major >= 7 && CUDA_VERSION < 9000)) {
+    std::stringstream err_string;
+    err_string << "PyTorch compiled with insufficient CUDA version: " 
+	       << CUDA_VERSION << " for the current GPU device " << prop.name 
+	       << " with device capability " << prop.major << "." << prop.minor;
```

Comment:
Things usually run just fine, it’s just nvrtc that fails, right? Maybe mention the cause of failure (tried to compiled fused function)